File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -8,50 +8,52 @@ documentation: UG
 
 # Troubleshooting and FAQ
 
-## BlinkBinaries are missing
+## Blink files are missing
 
 <table>
 <th style="font-size:14px" width="100px">Exception</th>
-<th style="font-size:14px">BlinkBinaries are missing</th>
+<th style="font-size:14px">Blink files are missing</th>
 <tr>
 <th style="font-size:14px" width="100px">Reason
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occur if the <i>'runtimes'</i> folder is not copied correctly from the NuGet folder.
 </td>
 </tr>
 <tr>
 <th style="font-size:14px" width="100px">Solution</th>
 <td>
-Set the path of the BlinkBinaries folder to the BlinkPath property of BlinkConverterSettings.
+Ensure that upon NuGet package installation and building, the 'runtimes' folder is automatically copied.
+<br/><br/>
+Please refer to the below screenshot,
+<br/><br/>
+<img src="htmlconversion_images/runtime_folder.png">
 <br/><br/>
 (Or)
 <br/><br/>
-Place the BlinkBinaries folder in bin folder of the project. 
+If you require explicit path setting, you can establish BlinkPath.
 <br/><br/>
-The BlinkBinaries will be available in the HTMLConverter installed location <span style="color:gray;font-size:14px"><i>($SystemDrive\Program Files (x86)\Syncfusion\HTMLConverter\xx.x.x.xx\BlinkBinaries)</i> </span>
-<br><br/>
-The BlinkBinaries also available in NuGet package installed location if you are using Blink HTML converter from NuGet packages. 
-
-</td>
-</tr>
-
-<tr>
-<th style="font-size:14px" width="100px">Mapping BlinkBinaries in Web Application</th>
-<td>
-{% highlight c# tabtitle="C#" %}
-
-//To refer BlinkBinaries from project location
-settings.BlinkPath = Server.MapPath("BlinkBinaries");
-
-//or
+Ex path: <i>C:\HtmlConversion\HTMl-to-PDF\HTMl-to-PDF\bin\Debug\net7.0\runtimes\win-x64\native\BlinkBinaries</i>
```

Comment:
C:\HtmlConversion\HTMl-to-PDF\HTMl-to-PDF\bin\Debug\net7.0\runtimes\win-x64\native\